utils/cb58/cb58_test.go

```diff
@@ -44,20 +46,14 @@ func TestEncodeDecode(t *testing.T) {
 	for _, test := range tests {
 		// Encode the bytes
 		strResult, err := Encode(test.bytes)
-		if err != nil {
-			t.Fatal(err)
-		}
+		require.NoError(err)
 		// Make sure the string repr. is what we expected
-		require.Equal(t, test.str, strResult)
+		require.Equal(test.str, strResult)
 		// Decode the string
 		bytesResult, err := Decode(strResult)
-		if err != nil {
-			t.Fatal(err)
-		}
+		require.NoError(err)
 		// Make sure we got the same bytes back
-		if !bytes.Equal(test.bytes, bytesResult) {
-			t.Fatal("bytes not symmetric")
-		}
+		require.True(bytes.Equal(test.bytes, bytesResult))
```

Comment:
Can't we use `require.Equal`?

Reply:
`bytes.Equal` has special handling around `nil` that `require.Equal` does not have.